EIPS/eip-template.md

```diff
@@ -0,0 +1,362 @@
+---
+title: New log filter data structure
```

Comment:
Rather than "new", is there a better description for the proposed log filter data structure?